galasa-ui/src/lib/data/testRuns.ts

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+// All imports related to data fetching move here
+import { ResultArchiveStoreAPIApi, Run, RunResults, UserData } from "@/generated/galasaapi";
+import { createAuthenticatedApiConfiguration } from "@/utils/api";
+import { CLIENT_API_VERSION, MAX_RECORDS, BATCH_SIZE } from "@/utils/constants/common";
+import { fetchAllUsersFromApiServer } from "@/app/users/page";
+
+/**
+ * The structure returned by the data fetching function.
+ */
+export interface TestRunsData {
+    runs: Run[];
+    limitExceeded: boolean;
+  }
+  
+  interface fetchAllTestRunsByPagingParams {
```

Comment:
Very minor - this `fetchAllTestRunsByPagingParams` interface seems to be indented by 2 spaces when it doesn't need to be